iree/turbine/kernel/wave/utils.py

```diff
@@ -811,6 +837,10 @@ def get_mfma_load_elems_per_thread(mfma_variant: MMAType) -> int:
             return 8
         case MMAType.F32_32x32x16_F8:
             return 8
+        case MMAType.F32_16x16x32_K4_F8:
```

Comment:
can we also reuse the case statements like 
```
case MMAType.F32_16x16x32_F8 | MMAType.F32_16x16x32_K4_F8
```

Reply:
done, thanks